server/tests/integration/commits.test.ts

```diff
@@ -99,4 +105,45 @@ describe('Commits endpoints', () => {
       );
     });
   });
+
+  describe('POST /commits', () => {
+    test('Should require customer auth', async () => {
+      await request(app).post('/commits');
+
+      expect(customerAuth).toHaveBeenCalled();
```

Comment:
what is the scope of the mock? will we get a new mock for each test?

if the mock is the same for all test cases, then we need to reset the mock in the setup of each test

Reply:
@ifhuang Oh right that slipped my mind! Adding it in now! The scope of the mock is for all the tests in the file.